lively.halos/morph.js

```diff
@@ -1449,7 +1455,9 @@ class CopyHaloItem extends HaloItem {
     const world = halo.world();
     const isMultiSelection = t instanceof MultiSelectionTarget;
     const origin = t.globalBounds().topLeft();
+    // the original morphs are needed so we can refocus them with a halo after copying
     const morphsToCopy = isMultiSelection ? t.selectedMorphs : [t];
+    const modified_morphsToCopy = morphsToCopy.filter(m => !(m instanceof CommentIndicator)).map(m => m.copy(true));
```

Comment:
Does this need to be explicit? We could also add a function mayBeCopied() to morph

Reply:
I don't like instanceof :(

Reply:
+1
I thought `copy` would return false in those cases and could be used to exclude those morphs

Reply:
And I am not sure what this line should do. What are "modified" morphs to copy? It looks to me like these are already the copies?